website/versioned_docs/version-0.76/getting-started.md

```diff
@@ -0,0 +1,140 @@
+
+## Create a new React Native project
+
+Remember to call `@react-native-community/cli init` from the place you want your project directory to live.
+
+<!-- Note, make sure both `@react-native-community/cli@ABC` and `--version "XYZ"` are pointing to the correct NPM tags in the command below. -->
+
+<!-- 1. For the next version (i.e. in docs/getting-started.md) use "next" for the CLI and "nightly" for the RN version -->
+<!-- 2. For the latest stable version in versioned_docs use "latest" for both the CLI and RN version -->
+<!-- 3. For older stable versions you'll have to look up the CLI version, but for the RN version use the stable tag name, i.e. "0.73-stable" -->
+
+<!-- See https://www.npmjs.com/package/@react-native-community/cli?activeTab=versions for the CLI version tags. -->
+<!-- See https://www.npmjs.com/package/react-native?activeTab=versions for the RN version tags. -->
+
+```bat
+npx --yes @react-native-community/cli@latest init <projectName> --version "latest"
+```
+
+### Navigate into this newly created directory
+
+React Native will have created your project in a new sub-directory, which you must enter before continuing.
+
+```bat
+cd <projectName>
+```
+
+### Add React Native Windows to your project's dependencies
+
+<!-- Note, make sure "version" is pointing to the correct react-native-windows NPM tag in the command below. -->
+
+<!-- 1. For the next version (i.e. in docs/getting-started.md) use "canary" -->
+<!-- 2. For other versions in versioned_docs use the version in the format "^0.XY.0" -->
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Using Yarn (Recommended)-->
+
+```bat
+yarn add react-native-windows@^0.76.0
+```
+
+<!--Using NPM-->
+
+```bat
+npm install --save react-native-windows@^0.76.0
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Initialize the React Native Windows native code and projects
+
+Lastly, initialize the React Native for Windows application with the [init-windows command](init-windows-cli.md):
+
+```bat
+npx react-native init-windows --overwrite
+```
+
+> **Note:** RNW templates contain a customized `metro.config.js` file, which is meant to merge cleanly into the default config provided by the standard React Native project template. If you are starting a new app, overwriting `metro.config.js` should have no impact. However, if you are adding Windows to an existing app with an already modified `metro.config.js` file, please make sure to back up and re-apply your modifications after adding RNW.
+
+> **Note:** Previous versions of the RNW Getting Started steps recommend the use of the `react-native-windows-init` command, which is being phased out. For documentation of that command, see: [React Native Windows Init CLI](https://microsoft.github.io/react-native-windows/init-cli).
```

Comment:
We can remove this paragraph now. Please also remove from `docs/getting-started.md`.